src/test/java/org/isf/permissions/test/PermissionIoOperationsTest.java

```diff
@@ -0,0 +1,61 @@
+package org.isf.permissions.test;
+import org.isf.permissions.model.GroupPermission;
+import org.isf.permissions.model.Permission;
+import org.isf.permissions.service.GroupPermissionIoOperationRepository;
+import org.isf.permissions.service.PermissionIoOperationRepository;
+import org.isf.permissions.service.PermissionIoOperations;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
The project standards do not support package level imports.   Please see the .ide-settings directory in this repo for import ordering and code formatting.